test/common_test/joint_features.cc

```diff
@@ -1092,8 +1095,8 @@ TYPED_TEST(JointFeaturesAttachDetachTest, JointAttachDetach)
         gz::math::eigen3::convert(frameDataModel1Body.linearVelocity);
       gz::math::Vector3d body2LinearVelocity =
         gz::math::eigen3::convert(frameDataModel2Body.linearVelocity);
-      EXPECT_NEAR(0.0, body1LinearVelocity.Z(), 1e-7);
-      EXPECT_NEAR(0.0, body2LinearVelocity.Z(), 1e-7);
+      EXPECT_NEAR(0.0, body1LinearVelocity.Z(), 1e-3);
+      EXPECT_NEAR(0.0, body2LinearVelocity.Z(), 1e-3);
```

Comment:
Since these constraints are being loosened, do you think it would also be helpful to print the actual quantiative errors somehow?  This would help in comparing engines.  These wouldn't be actual test expectations, but rather additional output for developers.

Reply:
https://github.com/gazebosim/gz-physics/issues/430